test/support/constants.py

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
Can we rename this `paths.py` or something?

Reply:
I actually intended to start putting things that aren't paths in there as well, like expected values for XML parser unit tests, etc. The ones in there now just happen to all be paths.

Reply:
:+1: